tests/net/gnrc_rpl/tests/01-run.py

```diff
@@ -162,8 +165,7 @@ def test_alternative_route(factory, zep_dispatch):
 
 
 def run_test(func, factory):
-    with Popen(['../../dist/tools/zep_dispatch/bin/zep_dispatch',
-                '-t', '-', '127.0.0.1', '17754'], stdin=subprocess.PIPE) as zep_dispatch:
+    with Popen([ZEP_DISPATCH_PATH, '-t', '-', '127.0.0.1', '17754'], stdin=subprocess.PIPE) as zep_dispatch:
```

Comment:
nice cleanup :)

Reply:
Hmm, it seems to be required in other places...